docs/dev/framework/caching.md

```diff
@@ -230,13 +230,28 @@ When working with DCAs in the Contao back end you don't have to register callbac
 tags are being invalidated. This is because Contao invalidates a certain set of tags whenever a back end entry is created,
 updated or deleted. The tags are as follows:
 
-* `contao.db.<table-name>`
 * `contao.db.<table-name>.<id>`
-* `contao.db.<parent-table-name>` (only if there is a parent table defined)
+* `contao.db.<table-name>` OR `contao.db.<parent-table-name>` (depending on whether a parent table is defined or not)
```

Comment:
Basically it goes up and down the tree recursively and adds a general tag for the top level.

So in case of page -> article -> content element if you edit article ID 42 you will end up having the following tags invalidated:

* `contao.db.tl_page` // General tag because there's no ptable for `tl_page`
* `contao.db.tl_page.<page_id>` // page ID = `tl_article.pid`
* `contao.db.tl_article.42`
* `contao.db.tl_content.<ce_id_1>` // All tl_content with ptable `tl_article` and `pid=42`
* `contao.db.tl_content.<ce_id_2>` // All tl_content with ptable `tl_article` and `pid=42`

Reply:
Thanks for your comment, will check on that 👍

Reply:
I addressed your comment in my last commit. The `tl_news` example was exactly like your article example, so that should be fine. I broke up the tags list in the record/table itself, parent and child tables and addressed the recursion, to make it more clear.